src/pages/index.vue

```diff
@@ -201,8 +201,9 @@ const HomePage = {
 
     const searchTerm = ref('')
     const handleSearch = async () => {
+      if (!searchTerm.value) return
```

Comment:
I wonder if we should `.trim()` the search term as well? Like if it's all spaces, do we still want to submit it?

DuckDuckGo actually displays an error when you try to submit a string of spaces.

<img width="775" alt="Captura de Tela 2022-02-08 às 05 32 18" src="https://user-images.githubusercontent.com/24264157/152969776-c9af6d0e-8961-421c-86db-754a3ee92d87.png">

Reply:
This issue is actually already handled by Vue with the `v-model`'s `.trim` modifier:
https://github.com/WordPress/openverse-frontend/blob/31ba45713247e31c9ae2c7bd251808d586b0a6e9/src/pages/index.vue#L45
I did go back and test it, and it does work, you can't search for ' '.

Reply:
😱 I should have tried it out before commenting!